source/js/components/share-button-group/share-button-group.jsx

```diff
@@ -0,0 +1,158 @@
+import React from "react";
+import classNames from "classnames";
+import copyToClipboard from "../../copy-to-clipboard";
+
+export default class ShareButtonGroup extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      linkCopied: false
+    };
+  }
+
+  componentDidMount() {
+    if (this.props.whenLoaded) {
+      this.props.whenLoaded();
+    }
+  }
+
+  renderFacebookButton() {
+    let label =
+      this.props.version === `mini` ? (
+        <span class="sr-only">Share on Facebook</span>
```

Comment:
Changed this from `Share by Facebook` to `Share on Facebook`. I can ping Kevin tomorrow to verify the wording.